### DISCUSSION TITLE
Cannot use dynamic routes in any Next version higher than 13.4.12

### DISCUSSION THREAD

**Discussion by sandervspl** [vercel/next.js#55951] [Help]
### Summary

We host our NextJS server on an AWS EC2 instance. For some reason we cannot upgrade Next to any version beyond 13.4.12. Any route with brackets (dynamic, catch-all, optional catch-all, etc.) will return as a 404 in the network tab. The files it is trying to download are `layout`, `loading`, and `page`. If I downgrade back to 13.4.12 I do not see these files being downloaded in the network tab any more and everything works as expected.

I tried hosting the app on Vercel and it worked fine there. Also on a Digitalocean instance everything works as expected.

Note that we do not have any `layout`, `loading`, or `page` that are client component. They are all server components.

<img width="1986" alt="Screenshot 2023-09-25 at 12 59 19" src="https://github.com/vercel/next.js/assets/4492636/1aad4d25-2e97-44ef-b434-88a98d344e9e">

I'm asking here because I do not have a way to reproduce this issue. I'm hoping someone has encountered something similar before or maybe knows what I can do next to try and debug this issue. Our sysops team does not think it is a EC2 config issue.

**Comment by tvoyammn** [comment]
Hey! Have you got any solution so far? We experiencing the same issue

**Reply by sandervspl** [reply]
We do not have a solution currently, no :(

**Reply by xllentech** [reply]
We are having similar issue, any version above 13.4.12 including 14+, dynamic routes get 404 with chunk load failed. App router, self-hosted using apache. Evething works as expected on 13.4.12.
